py/dml/messages.py

```diff
@@ -1511,6 +1511,21 @@ class EVLACONST(DMLError):
     fmt = ("variable length array declared with (partially) const-qualified "
            + "type")
 
+class ESIZEOFOP(DMLError):
```

Comment:
Verification revealed an interesting problem with this error wrapping: control flow paths that rely on catching errors that are not terminal for the program, in particular, the eager evaluation of params in `structure.py` does `except EIDXVAR` to detect when static indices are used. Do we want to ad-hoc patch such `except`s, or is this indicative that this entire approach is wrong?